AsyncSwift/Views/EventDetailView.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  EventDetailView.swift
+//  AsyncSwift
+//
+//  Created by Kim Insub on 2022/09/08.
+//
+
+import SwiftUI
+
+struct EventDetailView: View {
+
+    private let event: EventModel.Event
+    @StateObject var observed = Observed()
+
+    init(event: EventModel.Event) {
+        self.event = event
+    }
+
+    var body: some View {
+        ScrollView {
+            VStack(alignment: .leading, spacing: 0) {
+                customDivider
+                    .padding(.top, 10)
+                description
+                customDivider
+                information
+                Spacer()
+            }
+        }
+        .navigationTitle(event.detailTitle)
+        .confirmationDialog("", isPresented: $observed.showSheet, titleVisibility: .hidden) {
+            if let naverMapURL = URL(string: event.addressURLs.naverMapURL) {
+                Link("네이버 지도로 길 찾기", destination: naverMapURL)
+            }
+            if let kakaoMapURL = URL(string: "event.addressURLs.kakaoMapURL") {
```

Comment:
```suggestion
            if let kakaoMapURL = URL(string: event.addressURLs.kakaoMapURL) {
```

디버깅을 위해 수정된 것이 그대로 commit 된 것 같아요 .. !!